src/transform/ThemeTransform.js

```diff
@@ -41,14 +62,18 @@ const setTheme = (document, theme) => {
  * @return {void}
  */
 const classifyElements = element => {
-  Polyfill.querySelectorAll(element, 'img').forEach(image => {
-    if (!ElementUtilities.closestInlineStyle(image, 'background')) {
+  Polyfill.querySelectorAll(element, 'img')
+    .filter(imageNeedsWhiteBackground)
+    .forEach(image => {
       image.classList.add(CONSTRAINT.IMAGE_NO_BACKGROUND)
-    }
-    if (!ElementUtilities.isNestedInTable(image)) {
-      image.classList.add(CONSTRAINT.IMAGE_NONTABULAR)
-    }
-  })
+    })
+  /* en > Away colours > 793128975 */
+  /* en > Manchester United F.C. > 793244653 */
+  /* en > Pantone > 792312384 */
+  Polyfill.querySelectorAll(element, 'div.color_swatch div, div[style*="position: absolute"]')
```

Comment:
Ya I struggled with this - there were issues with trying to get `:not()` to work with multiple exceptions. Didn't seems to have widespread support.